pkg/controller/jobs/job/job_controller.go

```diff
@@ -292,7 +292,7 @@ func (j *Job) podsCount() int32 {
 func (j *Job) minPodsCount() *int32 {
 	if strVal, found := j.GetAnnotations()[JobMinParallelismAnnotation]; found {
 		if iVal, err := strconv.Atoi(strVal); err == nil {
-			return pointer.Int32(int32(iVal))
+			return ptr.To[int32](int32(iVal))
```

Comment:
Not sure on generics syntax but I did try `ptr.To[int32](ival)` but this was a compile error.  I had to cast.

Reply:
It's expected behavior since `iVal` typed is `int`.